languages/csharp/exercises/concept/nullability/.meta/Example.cs

```diff
@@ -0,0 +1,24 @@
+using System;
+
+public static class Badge
+{
+    public static int ComputeWidthPx(string? text,
```

Comment:
I like how the `text` parameter being nullable, we force people to deal with potential null-reference issues.